### PR TITLE
Added workflow files

### PR DESCRIPTION
**Pull request by talha0324**

### CODE REVIEW

**Comment by stakater-user** [comment]
@talha0324 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/c59cbfd6658137e145c9067142d194dff798577e/checks) has Failed!

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-68-1`

File: .github/workflows/pull_request.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+name: Pull Request
+
+on:
+  pull_request:
+    branches:
+      - 'master'
+    types:
+      - opened
+      - reopened
+      - synchronize
```
Comment (ahmedwaleedmalik) [inline .github/workflows/pull_request.yaml]: Only specify types explicitly if the configuration is different from the default. These are the default values.

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          fetch-depth: 0 # otherwise, you will fail to push refs to dest repo
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
```
Comment (ahmedwaleedmalik) [inline .github/workflows/release.yaml]:
```suggestion
          release_name: ${{ github.ref }}
```

**Review by ahmedwaleedmalik — CHANGES_REQUESTED** [review]

**Comment by ahmedwaleedmalik** [comment]
- Remove `JenkinsFile`
- You'll probably have to update `Dockerfile` as well.

**Comment by stakater-user** [comment]
@talha0324 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/55a8927e8541f90a8b6126817d526d5092a2fe13/checks) has Failed!

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/43474cc90168ded15dee5997053f46092a26b4f2/checks) has Failed!

**Review by ahmedwaleedmalik — APPROVED** [review]

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit//checks) has Failed!

**Pull request merged**